.github/workflows/ci.yml

```diff
@@ -469,92 +469,78 @@ jobs:
   #       run: |
   #         ./build_tools/cmake/ctest_all.sh ${BUILD_DIR}
 
-  ################################## Subsets ###################################
-  # Jobs that build some subset of IREE
+  ############################### Configurations ###############################
+  # Jobs that build IREE in some non-default configuration
   ##############################################################################
+
   build_test_runtime:
```

Comment:
I'm considering shortening this to just "runtime" so less text gets cut off: 
![image](https://github.com/iree-org/iree/assets/4010439/b469cc0d-af2d-4f3a-b345-9da98f8b55c7)

Reply:
Yeah that would be fine. I wonder why github repeats the matrix name inside the matrix when we can already see the header

Reply:
I'm on the fence about the naming, at least while there are so many other jobs not in matrices. Some jobs only build, and some jobs run tests using packages (so they aren't "building" the compiler portion of the project), so spelling out "build_test" here does help.

Might just continue to update the names as I refactor the rest of the workflow files.